modules/data-streams/build.gradle

```diff
@@ -14,7 +14,7 @@ esplugin {
 restResources {
   restApi {
     include 'bulk', 'count', 'search', '_common', 'indices', 'index', 'cluster', 'rank_eval', 'reindex', 'update_by_query', 'delete_by_query',
-      'eql', 'data_stream', 'ingest', 'cat'
+      'eql', 'data_stream', 'ingest', 'cat', 'capabilities'
```

Comment:
@thecoop @rjernst I wonder if we should add this by default for all YAML test projects as I assume it's necessary if you use any capabilities-based `requires` conditions?